src/index.ts

```diff
@@ -21,14 +25,7 @@ export function presetShadcn(options: PresetShadcnOptions = {}): Preset<Theme> {
 
           ${generateCSSVars(options)}
 
-          * {
-            border-color: hsl(var(--border));
-          }
-
-          body {
-            color: hsl(var(--foreground));
-            background: hsl(var(--background));
-          }
+          ${globals ? generateGlobalStyles() : ''}
```

Comment:
I think maybe we don't need this option. It’s always good to have this.

Reply:
Not sure about that, it's always better to allow more control, yesterday I was setting up `PrimeVue tailwind preset mode` + `Shadcn` using UnoCSS, and I was wondering what overridden my background color as I did set it in my `main.(s)css` file.
UnoCSS styles (`__uno.css`) is generated after Nuxt's `css` config and could introduce some priority headaches for user control.

Some image for reference:
![image](https://github.com/hyoban/unocss-preset-shadcn/assets/23612546/2eaf06e7-ad03-4c9c-b4eb-9f95c53f6c58)
![image](https://github.com/hyoban/unocss-preset-shadcn/assets/23612546/69a6dd9c-0ec0-4b9d-8475-c886b2412229)